python/grass/jupyter/setup.funcs.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import grass.script as gs
+import grass.script.setup as gsetup
+
+
+def _set_notebook_defaults():
+    """
+    This function sets several GRASS environment variables that are
+    important for GRASS to run smoothly in Jupyter. A complete list and
+    description of environment variables can be found here:
+    https://grass.osgeo.org/grass78/manuals/variables.html
+    """
+    # We want functions to raise exceptions and see standard output of
+    # the modules in the notebook.
+    gs.set_raise_on_error(True)
+    gs.set_capture_stderr(True)
+
+    # Allow overwrite of existing maps
+    os.environ["GRASS_OVERWRITE"] = "1"
+
+
+def init(path, location, mapset):
+    """
+    This function initiates a GRASS session and sets GRASS
+    environment variables.
+
```

Comment:
This might be a good place to mention that it will be overwriting the existing maps of the same name, i.e., as it happens with Python variables - ideally in some way which is 1) not confusing Python users and 2) still warns GRASS users that the default is different than what GRASS usually does. ... or you can put a `# TODO:` comment above the function to get to this later.

Reply:
I wrote a short sentence to that effect. I'm not sure if it's the clearest - I can put a `#TODO:` comment to clarify later too?

Reply:
Good enough for now.